distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Java_Table_Access.enso

```diff
@@ -0,0 +1,9 @@
+from Standard.Base import all
+
+## PRIVATE
+   Internal Enso use only. Allows access to the underlying java_table for modules outside of Standard.Table
+type Java_Table_Access
```

Comment:
Why don't we just do this as a static method.

I'd be tempted to add a `to_java_table` method into `Standard.Table.Table` as a static module method symmetric with `from_java_table`.

Reply:
Done. Much cleaner.